jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -4,10 +4,10 @@
 {{ $AUTH_TYPE := .Env.AUTH_TYPE | default "internal" -}}
 {{ $JICOFO_AUTH_TYPE := .Env.JICOFO_AUTH_TYPE | default $AUTH_TYPE -}}
 {{ $JICOFO_AUTH_LIFETIME := .Env.JICOFO_AUTH_LIFETIME | default "24 hours" -}}
-{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "0" | toBool -}}
+{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "1" | toBool -}}
```

Comment:
Upon further inspection I think we should keep this defaulting to disabled, for the time being. With the rest of the changes inplace you'll still be able to enable OCTO over SCTP though.

Reply:
Done in b2ecc27. :)